content/en/docs/concepts/storage/storage-classes.md

```diff
@@ -290,13 +290,17 @@ parameters:
   type: io1
   iopsPerGB: "50"
   encrypted: "true"
+  tagSpecification_1: "key1=value1"
+  tagSpecification_2: "key2=value2"
 allowedTopologies:
 - matchLabelExpressions:
   - key: topology.ebs.csi.aws.com/zone
     values:
     - us-east-2c
 ```
 
+- `tagSpecification`: Tags with this prefix are applied to dynamically provisioned EBS volumes. For more details, refer to the [AWS_EBS_CSI_Driver StorageClass Tagging](https://github.com/kubernetes-sigs/aws-ebs-csi-driver/blob/master/docs/tagging.md) documentation.
```

Comment:
- this doesn't have to be a list item;
- please wrap the long line manually, e.g.
```suggestion
`tagSpecification`: Tags with this prefix are applied to dynamically provisioned EBS volumes.
For more details, refer to [AWS_EBS_CSI_Driver StorageClass Tagging](https://github.com/kubernetes-sigs/aws-ebs-csi-driver/blob/master/docs/tagging.md).
```

Reply:
@tengqm I have applied the proposed changes. Please take another look.